### PR TITLE
Added tests to use scripts to perform rolling restart of a WebLogic domain or cluste

### PR DESCRIPTION
**Pull request by hzhao-github**
Use case:

1. Rolling restart the configured cluster using the sample script rollCluster.sh script
2. Rolling restart the dynamic cluster using the sample script rollCluster.sh script
3. Rolling restart the domain using the sample script rollDomain.sh script

Jenkins job:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4998 (parallel)

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java [hunk 1]
```diff
@@ -1257,6 +1402,27 @@ private void scalingClusters(String clusterName, String serverPodName, int repli
...
+    logger.info("managed server " + configServerPodName + " stopped successfully.");
+
+    assertDoesNotThrow(() ->
+        executeLifecycleScript(STOP_SERVER_SCRIPT, SERVER_LIFECYCLE, dynamicServerName),
+        String.format("Failed to run %s", STOP_SERVER_SCRIPT));
+    checkPodDeleted(dynamicServerPodName, domainUid, domainNamespace);
+    logger.info("managed server " + dynamicServerPodName + " stopped successfully.");
+  }
+
   private static void createDomainSecret(String secretName, String username, String password, String domNamespace) {
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java:1426]: Use the utility method oracle.weblogic.kubernetes.utils.CommonMiiTestUtils.createDomainSecret

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java:1424]: Changed

**Review by anpanigr — APPROVED** [review]
LGTM

**Review by vanajamukkara — APPROVED** [review]
LG

**Pull request merged**